Source/Rows/DatePickerRow.swift

```diff
@@ -58,7 +58,13 @@ open class DatePickerCell: Cell<Date>, CellType {
             #if swift(>=5.2)
                 if #available(iOS 14.0, *) {
                     #if swift(>=5.3) && !(os(OSX) || (os(iOS) && targetEnvironment(macCatalyst)))
+                    // UIDatePickerMode .countDownTimer is unsupported when using style .inline.
+                    if datePicker.datePickerMode == .countDownTimer {
```

Comment:
This has already been fixed in the master branch. Have a look at line 57

Reply:
Uuups, sorry. I fixed it in 5.3.2 for testing and then just applied my patch to the fork. :-/
Thanks and sorry for bothering you!

Reply:
No problem. Just released that fix